packages/mongo/tests/lib/MongoProvider.test.ts

```diff
@@ -1,1729 +1,5 @@
-
-				expect(typeof payload).toBe('object');
-
-				const { method, trigger, error, key, path, hook, data } = payload;
-
-				expect(method).toBe(Method.Update);
-				expect(trigger).toBeUndefined();
-				expect(error).toBeUndefined();
-				expect(key).toBe('test:update');
-				expect(path).toEqual([]);
-				expect(typeof hook).toBe('function');
-				expect(data).toBeUndefined();
-			});
-
-			test('GIVEN provider w/ data at key THEN returns payload w/ data AND updates value at key', async () => {
-				await provider.set({ method: Method.Set, key: 'test:update', path: [], value: 'value' });
-
-				const payload = await provider.update({ method: Method.Update, key: 'test:update', path: [], hook: (value) => value });
-
-				expect(typeof payload).toBe('object');
-
-				const { method, trigger, error, key, path, hook, data } = payload;
-
-				expect(method).toBe(Method.Update);
-				expect(trigger).toBeUndefined();
-				expect(error).toBeUndefined();
-				expect(key).toBe('test:update');
-				expect(path).toEqual([]);
-				expect(typeof hook).toBe('function');
-				expect(data).toBe('value');
-			});
-
-			test('GIVEN provider w/ data at path THEN returns payload w/ data AND updates value at path', async () => {
-				await provider.set({ method: Method.Set, key: 'test:update', path: ['path'], value: 'value' });
-
-				const payload = await provider.update({ method: Method.Update, key: 'test:update', path: ['path'], hook: (value) => value });
-
-				expect(typeof payload).toBe('object');
-
-				const { method, trigger, error, key, path, hook, data } = payload;
-
-				expect(method).toBe(Method.Update);
-				expect(trigger).toBeUndefined();
-				expect(error).toBeUndefined();
-				expect(key).toBe('test:update');
-				expect(path).toEqual(['path']);
-				expect(typeof hook).toBe('function');
-				expect(data).toBe('value');
-			});
-		});
-
-		describe('with values method', () => {
-			test('GIVEN provider w/o data THEN returns payload w/o data', async () => {
-				const payload = await provider.values({ method: Method.Values, data: [] });
-
-				expect(typeof payload).toBe('object');
-
-				const { method, trigger, error, data } = payload;
-
-				expect(method).toBe(Method.Values);
-				expect(trigger).toBeUndefined();
-				expect(error).toBeUndefined();
-				expect(data).toEqual([]);
-			});
-
-			test('GIVEN provider w/ data THEN returns payload w/ data', async () => {
-				await provider.set({ method: Method.Set, key: 'test:values', path: [], value: 'value' });
-
-				const payload = await provider.values({ method: Method.Values, data: [] });
-
-				expect(typeof payload).toBe('object');
-
-				const { method, trigger, error, data } = payload;
-
-				expect(method).toBe(Method.Values);
-				expect(trigger).toBeUndefined();
-				expect(error).toBeUndefined();
-				expect(data).toEqual(['value']);
-			});
-		});
-	});
-});
+runProviderTest<MongoProvider.Options>(MongoProvider, MongoProviderError, { collectionName: 'josh-test' });
```

Comment:
`collectionName` falls back to the name passed in `init()` so this isn't needed.

```suggestion
runProviderTest<MongoProvider.Options>(MongoProvider, MongoProviderError);
```